src/commcare_cloud/ansible/roles/kafka/templates/kafka-server.service.j2

```diff
@@ -2,7 +2,7 @@
 Description=Apache Kafka server (broker)
 Documentation=http://kafka.apache.org/documentation.html
 Requires=network.target remote-fs.target
-After=network.target remote-fs.target kafka-zookeeper.service
+After=network.target remote-fs.target zookeeper-server.service
```

Comment:
what is that change?

Reply:
`kafka-zookeeper` does not exist, I'm not sure if previously it was called that way when/if bundled with kafka, but the scope of that line is to make sure that `kafka-server` is started only after those other services (on the same machine), including now the correct `zookeeper-server`.